ar_transaction_changes.gemspec

```diff
@@ -2,6 +2,7 @@
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'ar_transaction_changes/version'
+require 'all_changed_attributes/version'
```

Comment:
This version isn't even used in this gemspec.  Why did you add it in the first place?